test/parallel_api/algorithm/alg.modifying.operations/fill.pass.cpp

```diff
@@ -97,6 +97,85 @@ test_fill_by_type(::std::size_t n)
 #endif
 }
 
+void test_empty_list_initialization_for_fill()
```

Comment:
We have a way to run a test for all policy types as seen in this same test file via `invoke_on_all_policies` and a struct like `test_fill`.  Perhaps we should use that pattern to remove some redundancy.

Reply:
After offline discussion, it sounds like this is purposeful to reduce complexity and increase transparency for finding errors in a very simple test.  I'm OK with that even with some code replication.